.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+insert_final_newline = true
+
+[*.go]
+indent_size = 8
+indent_style = tab
+
+[*.{md, html}]
+indent_size = 4
+indent_style = space
+
+[Makefile]
+indent_size = 4
+indent_style = tab
```

Comment:
It is crucial for `Makefile` to use tab. Otherwise any command written in it won't be able to be executed.